SPaaS - Service process started twice

```
Traceback (most recent call last):
  File "/usr/local/bin/scenario_player", line 10, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/scenario_player/main.py", line 222, in run
    service_process.start()
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 106, in start
    assert self._popen is None, 'cannot start a process twice'
AssertionError: cannot start a process twice
20190729T040334 Collecting logsTraceback (most recent call last):
  File "/usr/local/bin/scenario_player", line 10, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/scenario_player/main.py", line 222, in run
    service_process.start()
  File "/usr/local/lib/python3.7/multiprocessing/process.py", line 106, in start
    assert self._popen is None, 'cannot start a process twice'
AssertionError: cannot start a process twice
20190729T040334 Collecting logs
```

The SP starts the service process twice - don't do that.

--- a/scenario_player/main.py
+++ b/scenario_player/main.py
@@ -219,7 +219,7 @@ def run(
                 log.warning("Press q to exit")
                 while not ui_greenlet.dead:
                     gevent.sleep(1)
-            service_process.start()
+            service_process.stop()
         except ServiceProcessException:
             service_process.kill()
         finally:

